bin/ovs-offline

```diff
@@ -53,29 +54,42 @@ usage() {
     echo "  -h:     Print help"
     echo "  -w:     Set working directory. Also configurable via OVS_DBG_WORKDIR env var. Default: /tmp/ovs-offline-dbg"
     echo "  -d:     Set debug mode"
+    echo "  -p:     Manage containers using podman (default: Docker)"
 }
 
 error() {
     echo "Error: $@"
     exit 1
 }
 
+set_container_cmd() {
+    if [ ! -x "$(command -v docker)" ]; then
```

Comment:
It would be better to first check for the docker daemon, then `command -v podman`

This is because a common package installed is `podman-docker`, this is just an alias to `podman` so the  `command -v docker` test will return true when we really want podman.

All that to say we can't trust  `command -v docker` due to `podman-docker`

For example:
```
❯ command -v docker
/usr/bin/docker
❯ command -v podman
/usr/bin/podman
❯ cat /usr/bin/docker
#!/usr/bin/sh
[ -f /etc/containers/nodocker ] || \
echo "Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg." >&2
exec /usr/bin/podman "$@"
```

Reply:
Aahh I see, thank you, I hadn't considered that.

Question; if podman-docker emulates a docker cli, and returns true to 'command -v docker', wouldn't it work the same whether ovs-offline runs commands with docker or podman?

Currently it checks 'command -v docker', if true, it proceeds with 'docker run ...', which would then alias to podman commands?

Reply:
Yes basically, you just get the annoying ```Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.``` message haha.

It's also nice to know exactly what your working with to account for the subtle differences between docker and podman IMO

Reply:
Okay, that makes sense, thank you! Just pushed the change to check for podman-docker before defaulting to using docker